docs/templates/reachview/firmware-reflashing.mdx

```diff
@@ -187,7 +198,7 @@ Get the latest ReachView image version and unzip it:
 	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/common/app-step2-choose-usb-rs.png" style="width: 300px;" /></p>
 {%endif%}
 
-* Select ReachView firmware .img file and click on _Flash_
+* Download the proposed ReachView firmware .zip file or select it from your disk. Click on _Flash_
```

Comment:
@TatianaAndreeva  Does the following worth mentioning:
1. Flasher supports both `.img` and `.zip` files
2. If the user selects an old image from disk, the flasher will show a warning about obsolete firmware

Reply:
Sure, let's add this, too